### ISSUE TITLE
[polyclipping] build failure: windows release-only

### ISSUE THREAD

**Issue by fabiencastan** [microsoft/vcpkg#21176]
**Host Environment**
- OS: Windows
- Compiler: MSVC

**To Reproduce**
`./vcpkg install --triplet x64-windows-release polyclipping`
Using the `triplets\community\x64-windows-release.cmake`

**Failure logs** 
```
  CMake Error at ports/polyclipping/portfile.cmake:22 (file):
    file RENAME failed to rename
      D:/a/AliceVision/vcpkg/packages/polyclipping_x64-windows-release/debug/share/pkgconfig
    to
      D:/a/AliceVision/vcpkg/packages/polyclipping_x64-windows-release/debug/lib/pkgconfig
    because: The system cannot find the path specified.
```

**Additional context**
https://github.com/microsoft/vcpkg/commit/d8b9fa9a535de1d3b4effe8fca5c18f573af88e8#r59364968

**Comment by NancyLi1013** [comment]
Hi @fabiencastan

Thanks for posting this issue. This has fixed by PR #21023. Please try to update to the latest vcpkg to rebuild this.

**Comment by fabiencastan** [comment]
Thanks!
